tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -203,11 +203,47 @@ def config_mirror(self, peer_cluster, **kw):
         self.wait_for_status(poolname=poolname, health_pattern="OK")
         peer_cluster.wait_for_status(poolname=poolname, health_pattern="OK")
 
+    # configure initial mirroring steps
+    def initial_mirror_config(self, mirror1, mirror2, poolname, imagename, **kw):
+        """
+        Calls create_pool function on both the clusters,
+        creates an image on primary cluster and configure the mirroring
+        waits for image to be present in secondary cluster with replying status
+        Args:
+            **kw:
+            mirror1 - primary cluster
+            mirror2 - secondary cluster
+            poolname - name for pool to be created on primary and secondary cluster
+            imagename - name of images created on primary and secondary cluster
+            imagesize - size of the image created
+            mode - mirror mode to be configured pool or image
+            mirrormode - type of mirror configured journal or snapshot
+        """
+        mode = kw.get("mode")
```

Comment:
Can we induce parallelism in waiting for status/creating pool/image ?
(parallel module from ceph directory)

Reply:
we will be addressing this in new ticket. 
Marking it resolved as of now.